webrob/pages/QA.py

```diff
@@ -31,6 +32,7 @@ def render_QA_page():
 
 @app.route('/video')
 def render_video_page():
+    neem_manager = NEEM_Manager()
```

Comment:
this looks like an error. NEEM Manager is supposed to be a singleton with only one global instance.